src/requestforattestation/RequestForAttestation.ts

```diff
@@ -140,6 +140,35 @@ export default class RequestForAttestation implements IRequestForAttestation {
     })
   }
 
+  public static isIRequestForAttestation(
+    // ughh that function name... how do we want to call these typeguards?
```

Comment:
Given that it's a static and thus always called via `RequestForAttestation.isIRequestForAttestation`, we probably don't need the interface name? We could call them all something like `.isTypeOf()` or `typecheck()`. But it's true that the naming is tricky - we also would not want to convey that this is a type check for `RequestForAttestation`, because some methods may still be missing.

Reply:
We need a name that conveys basically: "is input of type RequestForAttestation Interface?"

So yeah naming is ofc always subject to change, will talk to @tjwelde about that